docs/site/translation.md

```diff
@@ -120,24 +63,25 @@ fvfixed date with single time (as was the case in CLDR 47 and earlier), then for
 
 ### Metazones
 
-There is a new metazone for Kazakhstan (which merged its two time zones).
+TBD
 
 ## Survey Tool
 
 Once trained and up to speed on [Critical reminders](#critical-reminders-for-all-linguists) (below), log in to the [Survey Tool](https://st.unicode.org/cldr-apps/) to begin your work.
 
 ### Survey Tool Changes
 
+1. The ability to search in the Survey Tool has been added in [CLDR-18423][] and supports searching for: values, English value, and for the codes
 1. There has been substantial performance work that will show up for the first time. If there are performance issues, please file a ticket with a row URL and an explanation for what happened.
```

Comment:
Is there a problem with the numbering? It seems like all the items start with "1". Or is that a feature?

Reply:
Usually in markdown you can use 1. for everything and it will be correctly numbered. LMK if it doesn't work in this markdown. It's a lot more convenient when things are likely to change numbers as list items are added/removed.